### DISCUSSION TITLE
Automatic Navigation Link Generation?

### DISCUSSION THREAD

**Discussion by Melvin8D** [godotengine/godot-proposals#7152] [3D]
A feature that I find missing in Godot that other engines seem to have is a way to automatically generate navigation links based off properties like Jump Height and Jump Distance and all that so you can have an easier way for objects to be able to jump up ledges and jump across gaps. I was wondering if Godot could add a feature like that if possible, as manually placing navigation links is pretty tedious.

**Comment by smix8** [comment]
> I was wondering if Godot could add a feature like that if possible, as manually placing navigation links is pretty tedious.

While possible no current plans to add it.

**Comment by AThousandShips** [comment]
How likely is it that these auto-generated links will be "good enough" that no or very minimal editing is required to fix them? How much would this reduce the work required to get a good result compared to having to add all links manually?

Would this feature be useful enough that people would regularly use it, justifying the (I assume) major work needed to implement it in core? If this feature isn't exact enough for most peoples' needs then it will go unused

On the flip side, how difficult would this be to implement as a plugin?

**Comment by suprohub** [comment]
Any updates???
